backend/infrahub/graphql/analyzer.py

```diff
@@ -639,7 +639,7 @@ def _populate_inline_fragment_node(
         self, node: InlineFragmentNode, query_node: GraphQLQueryNode
     ) -> GraphQLQueryNode:
         context_type = query_node.context_type
-        infrahub_model = self.schema_branch.get(name=node.type_condition.name.value)
+        infrahub_model = self.schema_branch.get(name=node.type_condition.name.value, duplicate=False)
```

Comment:
Good catch! I'm wondering about these methods where where we default to duplicate=True.. I'm wondering if we really want that to be the default.

Reply:
Also pushed this change (not sure if you saw it): https://github.com/opsmill/infrahub/pull/7341/files#diff-8fe408583ad84711bfe2ed274c9b892ab0b685027e716d716c375e5b64a69b04R116

Back in the time when we introduced `duplicate = True` for optimization, we wanted to selectively disable duplication to be safe and not introduce breakage.
I think we are at a point where we have more `duplicate=False` statements than the other way around, so we can probably switch to `duplicate=False` by default and use `duplicate=True` where it makes sense?

Reply:
Yeah we can have that as a tech debt item for Infrahub 1.6.

Reply:
Yes saw your other change as well and I think it highlights the problem of defaulting to False..